openssl/hkdf.go

```diff
@@ -19,6 +19,13 @@ type hkdf struct {
 	ctx *C.GO_EVP_PKEY_CTX
 }
 
+func SupportsHKDF() bool {
+	if openSSLVersion() < OPENSSL_VERSION_1_1_1 {
```

Comment:
We can eliminate the branch and just write `return openSSLVersion() >= OPENSSL_VERSION_1_1_1`.